examples/tasks/run-mock-psutil.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+# get the directory the script exists in
+__dir="$(cd ../../scripts/docker/large && pwd)"
+__proj_dir="$(cd ../../ && pwd)"
+
+export PLUGIN_SRC="${__proj_dir}"
+
+# verifies dependencies and starts influxdb
+. "${__proj_dir}/examples/tasks/.setup.sh"
+
+# start the influxdb container
+(cd $__dir && docker-compose up)
+
```

Comment:
Can we drop the user into a shell here so they can play with snap and the task that was just started.  See [this](https://github.com/intelsdi-x/snap-plugin-publisher-influxdb/blob/master/examples/tasks/run-mock-influxdb.sh#L21).